### ISSUE TITLE
Could I get pruned models on CIFAR10

### ISSUE THREAD

**Issue by dongxiao92** [yihui-he/channel-pruning#76]
Thanks for your impressive work.I'd like to know if I could find the pruned resnet model on CIFAR10.As I know, only models for imagenet are released now.
Thank you!

**Comment by Johnson-yue** [comment]
Yes, you can ,  you can train any dataset  !  you should modify  data_source： in  train.prototxt

**Comment by shi27feng** [comment]
Hi,
I have a similar question, I am trying to prune the CNN models in OpenPose, and their CNNs are trained by COCO, it seems there's no bgr.binaryproto provided by COCO, do I need to make it myself? Also, OpenPose seems to use lmdb format for data, how can I modify vgg.prototxt to adapt this (or does channel pruning support lmdb)?
Thank you in advance for the reply.
